api/v1beta1/charmedk8scontrolplane_types.go

```diff
@@ -107,6 +107,7 @@ type CharmedK8sControlPlaneStatus struct {
 
 	// Initialized is true when the target cluster has completed initialization such that at least once,
 	// the target's control plane has been contactable.
+	// +optional
```

Comment:
this was missing its optional specifier and was preventing patches from going through